### ISSUE TITLE
Incomplete Readme.md

### ISSUE THREAD

**Issue by NipunaPrashan** [oclif/color#2]
Fill the Usage section.

**Comment by youyiqin** [comment]
doc?

**Comment by kpayson** [comment]
Yes please give a few usage examples.  I would like to be able control output colors.  Can you show how to do that.

**Comment by Abhi347** [comment]
This repo doesn't look like it's maintained actively. I would suggest future readers to just use other npm libraries such as `chalk` (which is also used by this repo) for having colors in the logs. Additionally, the code is pretty small already, so you can just copy it and use it yourself, without depending upon another npm package which is not updating its dependencies regularly and thus may be a bottleneck for you to resolve the security vulnerabilities in your code.
